pages/cart.js

```diff
@@ -0,0 +1,104 @@
+const { I } = inject();
+
+module.exports = {
+  firstNameField: { xpath: '//input[@type="text"][@id="input-payment-firstname"]' },
+  lastNameField: { xpath: '//input[@type="text"][@id="input-payment-lastname"]' },
+  addressField: { xpath: '//input[@id="input-payment-address-1"]' },
+  cityField: { xpath: '//*[@id="input-payment-city"]' },
+  postcodeField: { xpath: '//*[@id="input-payment-postcode"]' },
+  countryDropdown: { xpath: '//a[@class="sbToggle"]' },
+  country: { xpath: '//li[a[text()="United States"]]' },
+  region_Dropdown: { xpath: '//div[@class="sbHolder"]/following::div[@class="col-sm-10"]' },
+  region_state: { xpath: '//li[a[text()="New York"]]' },
+  continueButton: { xpath: '(//input[@id="button-payment-address"])' },
+  continueStep2Button: { xpath: '(//div[@class="pull-right"]/input[@id="button-payment-address"])[1]' },
+  continueStep3Button: { xpath: '(//input[@id="button-shipping-address"])[1]' },
+  continueStep4Button: { xpath: '//input[@id="button-shipping-method"][1]' },
+  step5ConditionsAccepting: { xpath: '//input[@id="agree1"]' },
+  continueStep5Button: { xpath: '(//input[@id="button-payment-method"])[1]' },
+  flatShippingRate: { xpath: '//*[@id="collapse-checkout-confirm"]/div/div[1]/table/tfoot/tr[2]/td[2]' },
+  ecoTax: { xpath: '//*[@id="collapse-checkout-confirm"]/div/div[1]/table/tfoot/tr[3]/td[2]' },
+  vat: { xpath: '//*[@id="collapse-checkout-confirm"]/div/div[1]/table/tfoot/tr[4]/td[2]' },
+  total: { xpath: '//*[@id="collapse-checkout-confirm"]/div/div[1]/table/tfoot/tr[5]/td[2]' },
+  confirmOrderButton: { xpath: '//*[@id="button-confirm"]' },
+  successfulPurchaseText: { xpath: '//*[@id="content"]/h1/text()' },
+
+  fillCheckoutForm(customer) {
+    if (!this.firstNameField.value) {
```

Comment:
group all fill fields under single condition. Site does not display them randomly, specific condition for each is not required.